common-money.lic

```diff
@@ -28,8 +30,16 @@ module DRCM
     amount
   end
 
+  def hometown_currency(hometown_name)
+    return currency = get_data('town')[hometown_name]['currency']
+  end  
+
   def wealth(hometown)
-    currency = get_data('town')[hometown]['currency']
+    if $CURRENCIES.join("|").match /#{hometown}/i
```

Comment:
Does this mean we're calling `wealth` with an arg of a currency sometimes and a hometown sometimes?

Reply:
Yeah, there are a variety of places that use hometown name.  Either by calling wealth directly or indirectly via ensure_copper_on_hand, or other methods I haven't seen yet.

For example:

crossing-repair:
ensure_copper_on_hand(@repair_withdrawal_amount, settings.hometown)

deposit.lic:
copper_on_hand = wealth(hometown)

Reply:
Could we have two methods (and the one that looks up your hometown currency could call the other one)?

Reply:
Err... deposit.lic is not a dependency script. it was a copy of something I was using before.  If that matters.  It may be easier to clean up the scripts that call the method instead.